invenio_rdm_records/oai.py

```diff
@@ -27,6 +30,46 @@ def dublincore_etree(pid, record):
     return DublinCoreXMLSerializer().serialize_object_xml(record["_source"])
 
 
+def datacite_etree(pid, record):
+    """Dump the record (search result) into a DataCite XML format for OAI-PMH."""
+    data_dict = DataCite43XMLSerializer().dump_one(record["_source"])
+    return schema43.dump_etree(data_dict)
+
+
+def oai_datacite_etree(pid, record):
+    """Dump the record (search result) into a OAI DataCite XML format for OAI-PMH."""
+    datacentre_symbol_text = None
```

Comment:
- This is one of the things that should be configurable. For Zenodo e.g. that is [`CERN.ZENODO`](https://zenodo.org/oai2d?verb=ListRecords&metadataPrefix=oai_datacite), which is our "datacenter symbol" on DataCite (see from [their REST API](https://api.datacite.org/clients/cern.zenodo)).
- The variable itself also doesn't need to be initialized up here, but just directly get set below at `datacentre_symbol.text = current_app.config.get('DATACITE_DATACENTER_SYMBOL', None)`.
- For the config name, there's already a [`DATACITE_` set of variables used](https://github.com/inveniosoftware/invenio-rdm-records/blob/436bd2f4f6a75713db87ebf19d9cb962176d3899/invenio_rdm_records/ext.py#L105-L110), which we might actually piggy-back on.

Reply:
fixed